app/src/org/commcare/dalvik/activities/EntitySelectActivity.java

```diff
@@ -150,15 +150,9 @@ public void onCreate(Bundle savedInstanceState) {
             mResultIsMap = savedInstanceState.getBoolean(EXTRA_IS_MAP, false);
         }
         
-        try {
-            asw = CommCareApplication._().getCurrentSessionWrapper();
-            session = asw.getSession();
-        } catch(SessionUnavailableException sue){
-            //The user isn't logged in! bounce this back to where we came from
-            this.setResult(Activity.RESULT_CANCELED);
-            this.finish();
-            return;
-        }

```

Comment:
This bounce back shouldn't be needed now that the login screen is automatically launched when the session expires.